tests/test_threading.py

```diff
@@ -168,6 +175,7 @@ def test_model_context(self):
 
 
 # Same as TestLockedTransition but with LockedHierarchicalMachine
+@skipIf(pgv is None, 'AGraph diagram requires pygraphviz')
```

Comment:
afaik, no graph functionality is tested here. Instead of `skipIf` you can just remove `graph=True` from `get_predefined` in line 176(184). This way we do not have to pull pygraphviz into test_threading.py.